docker/README.md

```diff
@@ -0,0 +1,156 @@
+```console
+$ ./scripts/infrastructure.sh
+$ ./scripts/services.sh
+```
+
+You can also run individual services. For example:
+```console
+$ docker-compose up frontend # localhost:8080
+```
+
+If you ask to bring up a service with a dependency, Compose will treat this
+command as an implicit request to bring up the dependencies before bringing
+up the dependant. If you don&rsquo;t want that, use:
+
+```console
+$ docker-compose up --no-deps frontend
+```
+
+If Docker doesn&rsquo;t have the images it needs, it will automatically
+pull them. However, it won&rsquo;t pull existing images. If you want to run
+the latest versions of services, remember to ask for them first:
+
+```console
+$ docker-compose pull
+$ docker-compose up
+```
+
+## Troubleshooting
+### Unhealthy containers
+
+Docker Compose is configured so that containers that depend on other containers
+don't get started until their dependencies are not only running, but also ready
+for use.
+
+If a container doesn't start working after a certain period of time, it will be
+marked as &ldquo;unhealthy&rdquo; and Docker Compose will exit.
+
+The error mesage will look something like this:
+
+```console
+ERROR: for frontend  Container "fe34db64b428" is unhealthy.
+ERROR: Encountered errors while bringing up the project.
+```
+
+The hexadecimal number is the problematic container's ID.
+You can check to see which container is misbehaving with:
+
+```console
+$ docker container ls
+$ docker container logs fe34db64b428
+```
+
+(If you&rsquo;re new to Docker, the duration under &ldquo;STATUS&rdquo; is
+generally more applicable here than the one under &ldquo;CREATED&rdquo;.)
+
+This error is often due the process simply taking longer than usual to start,
+and waiting a bit longer is enough to resolve the issue.
+
+However, if you grow tired of waiting, or the logs look strange, you can usually
+fix the problem by restarting the container:
+
+```console
+$ docker container restart fe34db64b428
+```
+
+Once the container is marked as &ldquo;healthy&rdquo; you can just pick the
+the Docker Compose startup from where you left off:
+
+```console
+$ docker-compose up
+```
+
+For more serious problems you might need to tear down the environment with
+`docker-compose down` and start again.
+
+### Missing SQL tables
+
+The info-service doesn&rsquo;t come with its schema, so if you&rsquo;re setting
+up your environment for the first time your info-service you&rsquo;ll need to
+do them manually.
+
+The info-service schema isn&rsquo;t currently available publicly, and is stored
+in a private repository on GitHub. Request access from an existing team member.
+If you&rsquo;re working in Government but aren&rsquo;t a member of the DECS
+development team, ask someone from the Home Office in the cross-Government Slack
+channel to ask us on your behalf. (We might not be able to give you full access,
+but should be able to send you the basic schemas.)
```

Comment:
Maybe remove the last line here - we can evaluate on a case by case basis rather than setting expectations too soon.